simulation/README.md

```diff
@@ -7,15 +7,6 @@ Additionally, it publishes a [TwistStamped](http://docs.ros.org/melodic/api/geom
 ### Assumptions
 This readme assumes that you are able to mount the git repo in the docker container, and that you have the most recent Docker image pulled. 
 
-### Setting up the Docker Container
-You will need a Docker container with port 8080 forwarded. Additionally, you will need to use the simulation tag for the Docker container. Run the command that you normally do to create your Docker container according to the [documentation repo](https://github.com/DukeRobotics/documentation/tree/master/docker), but with `-p 8080:8080` added, and with `:simulation` at the end.
-    
-For example, you might run the command:
-
-`docker run -td -p 2200:2200 -p 8080:8080 --mount type=bind,source=C:\Users\Eric\Documents\Robotics\CS,target=/home/duke/dev/robosub-ros/src dukerobotics/robosub-ros:simulation`
-    
-[Here's](https://github.com/DukeRobotics/documentation/tree/master/docker) the link to the documentation repo with the Docker commands, if you need to look yours up.
-
 ### Downloading CoppeliaSim on your personal computer
 #### Downloading on Windows
 Download the installer for the Edu version from [this link](http://coppeliarobotics.com/winVersions), and run the downloaded .exe file.
```

Comment:
The heading below should read "Downloading on Ubuntu" since the downloads page only lists Ubuntu, not Linux in general.